etc/spack/defaults/darwin/packages.yaml

```diff
@@ -21,11 +21,14 @@ packages:
     - gcc
     - intel
     providers:
-      elf: [libelf]
-      unwind: [apple-libunwind]
+      elf:
+      - libelf
+      unwind:
+      - apple-libunwind
```

Comment:
Why change from `[...]` to `- ...` in `darwin/packages.yaml` but not `packages.yaml`?

Reply:
This file was updated automatically with a Spack command, so I guess the `- ...` is the default representation `spack_yaml` use for a Python list.